internal/integration/client_side_encryption_test.go

```diff
@@ -660,12 +660,83 @@ func TestFLE2CreateCollectionWithAutoEncryption(t *testing.T) {
 		encryptedClient.Database("db").CreateCollection(context.Background(), "coll", options.CreateCollection().SetEncryptedFields(encryptedFields))
 
 		// Check resulting events sent.
-		assert.Equal(mt, startedCommands, []string{
+		assert.Equal(mt, []string{
 			"create",          // Create ESC collection.
 			"create",          // Create ECOC collection.
 			"create",          // Create 'coll' collection.
 			"listCollections", // Run listCollections when processing `createIndexes` command for automatic encryption.
 			"createIndexes",
-		})
+		}, startedCommands)
+	})
+}
+
+func TestFLEIndexView(t *testing.T) {
```

Comment:
@kevinAlbs Would this be a good candidate as an optional FLE prose test for drivers that support index view?

Reply:
libmongocrypt [bypasses](https://github.com/mongodb/libmongocrypt/blob/6b328b46d2489526b7cc013a9b3d36eddfe3ab62/src/mongocrypt-ctx-encrypt.c#L2775) `listIndexes` and `dropIndexes`. The commands are returned as-is since the commands are not expected to contain values needing encryption.

I still agree with this PR. I think it is slightly preferable to process these commands with libmongocrypt for consistency with other commands. But I expect it is not strictly necessary, since the caller would not observe a difference when processing with libmongocrypt.

Since I expect is no observable behavior difference when processing `listIndexes` and `dropIndexes` with libmongocrypt, and a prose test may rely on private API that is not consistent among all drivers, I recommend against pursuing a prose test for all drivers.